src/commands/cloud-run/flare.ts

```diff
@@ -462,21 +482,38 @@ export const summarizeConfig = (config: IService) => {
  * @param logClient Logging client
  * @param serviceId
  * @param location
+ * @param startMillis start time in milliseconds or undefined if no start time is specified
+ * @param endMillis end time in milliseconds or undefined if no end time is specified
  * @param severityFilter if included, adds the string to the filter
  * @returns array of logs as CloudRunLog interfaces
  */
-export const getLogs = async (logClient: Logging, serviceId: string, location: string, severityFilter?: string) => {
+export const getLogs = async (
+  logClient: Logging,
+  serviceId: string,
+  location: string,
+  startMillis: number | undefined,
+  endMillis: number | undefined,
+  severityFilter?: string
```

Comment:
Isn't this the same?
```suggestion
  startMillis?: number,
  endMillis?: number,
  severityFilter?: string
```

Reply:
The only difference is the second option `startMillis?` doesn't require us the pass in any argument. What we currently have requires us to explicitly pass something in, even if what we pass in is `undefined`.
Since we always pass in `startMillis` and `endMillis` (and just pass in undefined when the flags aren't specified), I figured the current method header is better. 
WDYT?